emission/analysis/plotting/composite_trip_creation.py

```diff
@@ -152,3 +153,15 @@ def get_locations_for_confirmed_trip(ct, max_entries=100):
         sample_rate = len(locations)//max_entries + 1
         locations = locations[::sample_rate]
     return locations
+
+def get_sections_for_confirmed_trip(ct):
+    if "cleaned_trip" not in ct["data"]:
+        return [] # untracked time has no sections
+    trip_tl = esdt.get_cleaned_timeline_for_trip(ct["user_id"], ct["data"]["cleaned_trip"])
```

Comment:
I am not sure we always want to return `cleaned_section`s here. If we are only going to pick one, we should return the `inferred_section`s or ideally read it from the dynamic config.

In the old-style geojson object return, this was already a config option, but as part of the static config 
https://github.com/e-mission/e-mission-server/blob/cd0cdf9743aa7427e231fa95d887c5f1b156bf35/conf/analysis/debug.conf.json.sample#L10